Subject:  Use abspath for tbb linking
Patch author:  Orson Teodoro
Date:  Thu Aug 11 10:57:47 PDT 2022

This fixes undefined reference errors

ld: src/LinearMath/libLinearMath.so.3.24: undefined reference to `tbb::interface5::internal::task_base::destroy(tbb::task&)'

--- a/CMakeLists.txt.orig	2022-08-11 05:59:54.067375108 -0700
+++ b/CMakeLists.txt	2022-08-11 10:48:44.371278757 -0700
@@ -272,8 +272,8 @@ ENDIF (BULLET2_USE_OPEN_MP_MULTITHREADIN
 IF (BULLET2_USE_TBB_MULTITHREADING)
     SET (BULLET2_TBB_INCLUDE_DIR "not found" CACHE PATH "Directory for Intel TBB includes.")
     SET (BULLET2_TBB_LIB_DIR "not found" CACHE PATH "Directory for Intel TBB libraries.")
-    find_library(TBB_LIBRARY tbb PATHS ${BULLET2_TBB_LIB_DIR})
-    find_library(TBBMALLOC_LIBRARY tbbmalloc PATHS ${BULLET2_TBB_LIB_DIR})
+    find_library(TBB_LIBRARY tbb PATHS ${BULLET2_TBB_LIB_DIR} NO_CMAKE_SYSTEM_PATH)
+    find_library(TBBMALLOC_LIBRARY tbbmalloc PATHS ${BULLET2_TBB_LIB_DIR} NO_CMAKE_SYSTEM_PATH)
     ADD_DEFINITIONS("-DBT_USE_TBB=1")
     INCLUDE_DIRECTORIES( ${BULLET2_TBB_INCLUDE_DIR} )
     LINK_LIBRARIES( ${TBB_LIBRARY} ${TBBMALLOC_LIBRARY} )
